mmv1/third_party/terraform/services/workstations/go/resource_workstations_workstation_config_test.go.tmpl

```diff
@@ -1333,22 +1333,22 @@ func TestAccWorkstationsWorkstationConfig_vmTags(t *testing.T) {
 }
 
 func testAccWorkstationsWorkstationConfig_vmTags(context map[string]interface{}) string {
-	return acctest.Nprintf(`
-  data "google_project" "project" {
-    provider = "google-beta"
-  }
-  
-  resource "google_tags_tag_key" "tag_key1" {
-    provider = google-beta
-    parent = "projects/${data.google_project.project.number}"
-    short_name = "tf_test_tag_key1%{random_suffix}"
-  }
-  
-  resource "google_tags_tag_value" "tag_value1" {
-    provider = google-beta
-    parent = "tagKeys/${google_tags_tag_key.tag_key1.name}"
-    short_name = "tf_test_tag_value1%{random_suffix}"
-  }
+return acctest.Nprintf(`
+data "google_project" "project" {
+  provider = google-beta
+}
+
+resource "google_tags_tag_key" "tag_key1" {
+  provider = google-beta
+  parent = "projects/${data.google_project.project.number}"
+  short_name = "tf_test_tag_key1%{random_suffix}"
```

Comment:
I left short_names unchanged here, as not used in combination with the generated `testAccWorkstationsWorkstationConfig_workstationConfigBasicExample` function.

`testAccWorkstationsWorkstationConfig_update` does need to be changed, as that is used in combination with `testAccWorkstationsWorkstationConfig_workstationConfigBasicExample`